File: ljh/src/level1/java/SumBetweenTwoNumber.java
package level1.java;

/*
 * 프로그래머스 - 두 정수 사이의 합
 */
class SumBetweenTwoNumber {
    public long solution(int a, int b) {
        long answer = 0;
        long max = b;
        long min = a;

        if(a > b){
            max = a;
            min = b;
        }

        for(long i=min; i<=max; i++){
            answer += i;
        }

        return answer;
    }
}

File: ljh/bin/level1/js/SumBetweenTwoNumber.js
let a = 3;
let b = 5;
let answer = solution(a, b);
console.log(answer);

function solution(a, b) {
    var answer = 0;
    let max = b;
    let min = a;
    
    if(a>b){
        max = a;
        min = b;
    }
    for(let i=min; i<=max; i++){
        answer += i;
    }
    
    return answer;
}